app/src/main/java/ru/firstSet/kotlinOne/worker/OneTimeWorker.kt

```diff
@@ -0,0 +1,54 @@
+package ru.firstSet.kotlinOne.worker
+
+import android.content.Context
+import android.util.Log
+import androidx.work.*
+import org.koin.core.component.KoinApiExtension
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+class OneTimeWorker(val context: Context, params: WorkerParameters) : Worker(context, params) {
```

Comment:
@kamar19 Why do we need this `Worker`? We just need to start `CoroutineWorker` with periodic request and that is enough for us.